library/system/src/lib/yast2/execute.rb

```diff
@@ -62,6 +63,7 @@ def popup_error(&block)
         block.call
       rescue Cheetah::ExecutionFailed => e
         Yast.import "Report"
+        textdomain "base"
```

Comment:
NTH: I'd put the `textdomain` call into the main initialization code so you do not need to add another `textdomain` call later when adding a new translatable string somewhere else...